test/models/data_center_host_test.rb

```diff
@@ -139,6 +139,12 @@ def setup
     assert_includes @data_center_host.validator_score_v1s, validator2.validator_score_v1
   end
 
+  test "relationship has_many gossip_nodes returns correct nodes" do
```

Comment:
It's a bit misleading, the description says gossip_nodes and below you check gossip_node. PLease create at least two if it's has_many relationship